e3sm_diags/driver/qbo_driver.py

```diff
@@ -411,12 +411,11 @@ def _get_psd_from_wavelet(data: np.ndarray) -> Tuple[np.ndarray, np.ndarray]:
     Tuple(np.ndarray, np.ndarray)
         The period and PSD arrays.
     """
-    deg = 6
+    
     period = np.arange(1, 55 + 1)
-    widths = deg / (2 * np.pi / period)
+    widths = period
```

Comment:
I tried to play with pywt.cwt a little bit more, the only way to specify period is to use the values of period as widths/scales. Here is what I get compare to original plots:
with `pywt`:
![image](https://github.com/user-attachments/assets/e3e421a6-e7f7-4ed9-90fd-9a4725168103)
with `scipy`:
![image](https://github.com/user-attachments/assets/5b56d608-ec6a-47c1-82b4-4f4fc0344730)

My understand we don't really care about matching variance between two versions of code. @whannah1 can you advice if the code change is reasonable?

Reply:
I'm not sure I understand why you made that change in how the widths are specified, but to your question about variance, as long as both obs and model are treated consistently then we only need to know the relative different in the peaks. The absolute value of the peak height generally isn't useful for assessing/tuning the QBO. I'm sure one of the plots above is technically more "correct", but I have no idea how to figure which one that is.

Reply:
I read about cwt function from `scipy` and `pywt`, the fundamental equation are slightly different, and the input widths/scale, the only way to maintain an evenly spaced x axis (peirod), as in the scipy approach, is to use evenly space scale. Based on what I read, it is pretty controversial as which cwt is more accurate. I think we can use this version for now, as it produces good match with the FFT results on peak periods?

Reply:
Yes, I agree. I think either version fits our current needs despite the accuracy issue. The issue of even spacing is also not that critical as long as we have sufficient "resolution" of the peak period to generally distinguish a different in period of ~2 months or more. I feel that we have everything we need from the current method. I say we get this merged quickly to address the library deprecation issue and revisit the accuracy of the metric in the future as needed.

Reply:
Sounds great! Merging now. Thank you @whannah1.